sdk/eventhub/event-hubs/test/internal/sender.spec.ts

```diff
@@ -355,123 +357,136 @@ describe("EventHub Sender", function(): void {
       resetTracer();
     });
 
-    it("will not instrument already instrumented events", async function(): Promise<void> {
-      const { tracer, resetTracer } = setTracerForTest();
+    function legacyOptionsUsingSpanContext(rootSpan: TestSpan): Pick<TryAddOptions, "parentSpan"> {
+      return {
+        // @ts-ignore Using the deprecated field for testing
+        parentSpan: rootSpan.context()
+      };
+    }
 
-      const rootSpan = tracer.startSpan("test");
+    function legacyOptionsUsingSpan(rootSpan: TestSpan): Pick<TryAddOptions, "parentSpan"> {
+      return {
+        // @ts-ignore Using the deprecated field for testing
+        parentSpan: rootSpan
+      };
+    }
 
-      const list = [
-        { name: "Albert" },
-        {
-          name: "Marie",
-          properties: {
-            [TRACEPARENT_PROPERTY]: "foo"
+    function modernOptions(rootSpan: TestSpan): OperationOptions {
+      return {
+        tracingOptions: {
+          spanOptions: {
+            parent: rootSpan.context()
           }
         }
-      ];
+      };
+    }
 
-      const eventDataBatch = await producerClient.createBatch({
-        partitionId: "0"
-      });
+    [legacyOptionsUsingSpan, legacyOptionsUsingSpanContext, modernOptions].forEach((optionsFn) => {
+      describe.only(`tracing (${optionsFn.name})`, () => {
```

Comment:
```suggestion
      describe(`tracing (${optionsFn.name})`, () => {
```
The linter should check this *waits on linter*

Reply:
Oh man. Fixed. 

Also, while I was fixing that I found out I ran afoul of the other linting rule - don't use ts-ignore. I've added a directive for just that line so I can use it now.